tsconfig.json

```diff
@@ -10,6 +10,6 @@
     "sourceMap": true,
     "outDir": "./dist",
     "baseUrl": "./",
-    "incremental": true
+    "incremental": true,
```

Comment:
I don't think this is standards-compliant ([RFC8259](https://tools.ietf.org/html/rfc8259) page 6).

Reply:
fixed